fips-check.sh

```diff
@@ -18,19 +18,8 @@
 
 Usage() {
     cat <<usageText
-Usage: $0 [flavor [keep]]
+Usage: $0 flavor [keep]
 Flavor is one of:
-    linux (default)
-    ios
-    android
-    windows
-    freertos
-    openrtos-3.9.2
-    linux-ecc
-    netbsd-selftest
```

Comment:
oddly, `testing/jenkins-scripts/stable/PRB/fips/PRB-CAVP-selftest/PRB-CAVP-selftest.sh` is doing
```
./fips-check.sh netbsd-selftest
```
I assume the `-v2` version has the same thing.  Needs to be updated to get the PR to pass.

@kaleb-himes might have some insight on the history there, though he doesn't have blame on the script.

Reply:
Yup. Already on it. I was overzealous with the removal of things from that script